src/Main.java

```diff
@@ -1,5 +1,117 @@
+        int luda = 23;
+        System.out.println("У Людмилы Павловны — " + luda+ " ученика,");
+        int anna = 27;
+        System.out.println("у Анны Сергеевны — " + anna+ " учеников?");
+        int katya = 30;
+        System.out.println("У Екатерины Андреевны — " + katya+ " ученика.");
+        int totalNumberOfStudents = (luda + anna + katya);
+        System.out.println("Всего " + totalNumberOfStudents + " учеников");
+        int addSheetsPaper = 480;
+        System.out.println("Закуплено " + addSheetsPaper + " листов бумаги на все три класса");
+        int paperOneStudent = (addSheetsPaper/totalNumberOfStudents);
+        System.out.println("На каждого ученика рассчитано по " + paperOneStudent + " листов бумаги.");
+        System.out.println();
+        System.out.println("Задача 4");
+        int fabricated1Minute = 16 / 2;
+        System.out.println("Средняя производительность машины равна " + fabricated1Minute + " бутылок в 1 минуту");
+        int timePeriodOfOneInMinutes = 20;
+        int timePeriodOfSecondInMinutes = 24 * 60;
+        int timePeriodOfThreeInMinutes = 3 * timePeriodOfSecondInMinutes;
+        int timePeriodOfFourInMinutes = 30 * timePeriodOfSecondInMinutes;
+        int fabricated20Minutes = fabricated1Minute * timePeriodOfOneInMinutes;
+        System.out.println("За 20 минут машина произвела " + fabricated20Minutes +" штук бутылок,");
+        int fabricated1Days = timePeriodOfSecondInMinutes * fabricated1Minute;
+        System.out.println("За 1 день машина произвела " + fabricated1Days +" штук бутылок,");
+        int fabricated3Days = timePeriodOfThreeInMinutes * fabricated1Minute;
+        System.out.println("За 3 дня машина произвела " + fabricated3Days +" штук бутылок,");
+        int fabricated30Days = timePeriodOfFourInMinutes * fabricated1Minute;
+        System.out.println("За 1 месяц машина произвела " + fabricated30Days +" штук бутылок,");
+        System.out.println();
+        System.out.println("Задача 5");
+        int cansOfWhitePaitInOneClass = 2;
+        int cansOfBrownPaitInOneClass = 4;
+        int totalCans = 120;
+        int oneSetOfCans = (cansOfWhitePaitInOneClass + cansOfBrownPaitInOneClass);
+        int numberClass = (totalCans / oneSetOfCans);
+        int totalWhiteCans = (numberClass * cansOfWhitePaitInOneClass);
+        int totalBrownCans = (numberClass * cansOfBrownPaitInOneClass);
+        System.out.println("В школе, где " + numberClass+" классов, нужно " + totalWhiteCans+" банок белой краски и " + totalBrownCans+ " банок коричневой краски.");
+        System.out.println();
+        System.out.println("Задача 6");
+        int bananaWeigthInGramms = 80;
+        int milkWeigthInGramms = 105;
+        int iceCreamWeigthInGramms = 100;
+        int eggWeigthInGramms = 70;
+        int bananaPieces = 5;
+        int milkPieces = 2;
+        int iceCreamPieces = 2;
+        int eggPieces = 4;
+        int BananaWeightForAServing = bananaPieces * bananaWeigthInGramms;
```

Comment:
```suggestion
        int bananaWeightForAServing = bananaPieces * bananaWeigthInGramms;
```
названия переменных надо писать всегда с маленькой буквы